.github/workflows/acceptance-tests.yml

```diff
@@ -11,6 +11,7 @@ on:
         type: string
   pull_request: # you can run a specic job in your PR using GitHub labels
     types: [ labeled ]
+  merge_group: # needed for merge queue
```

Comment:
Not strictly related to these changes, but I see from latest build (https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/6745326427) and it seems merge_group does not work well with `dorny/paths-filter@v2`. I think we had a similar issue in cloud formation.

Reply:
the problem is that code health is calling the migration test action which it fails in the merge queue 🤔

Reply:
found a possible fix in https://github.com/PyO3/pyo3/pull/3212

Reply:
checking the merge queue

Reply:
interesting fix, nice find